components/layout/layout_task.rs

```diff
@@ -170,7 +170,7 @@ pub struct LayoutTask {
     pub id: PipelineId,
 
     /// The URL of the pipeline that we belong to.
-    pub url: Url,
+    pub url: RefCell<Option<Url>>,
```

Comment:
Why is this an `RefCell<Option<Url>>`, as opposed to just `RefCell<Url>`?  It looks like the code never stores None into the variable.